modules/libSceLibcInternal/entry.cpp

```diff
@@ -93,6 +93,14 @@ EXPORT SYSV_ABI int __NID(strncmp)(const char* str1, const char* str2, size_t si
   return std::strncmp(str1, str2, size);
 }
 
+EXPORT SYSV_ABI int __NID(strncpy)(char* dst, char* src, int len ) {
+  return Ok;
+}
+
+EXPORT SYSV_ABI int __NID(strcmp)(char* a, char* b) {
+  return Ok;
+}
+
```

Comment:
You can implement those actually, it's not that hard, look at the surrounding functions and do the same thing